partiql-ast/api/partiql-ast.api

```diff
@@ -1,5 +1,10 @@
 public final class org/partiql/ast/Ast {
 	public static final field INSTANCE Lorg/partiql/ast/Ast;
+	public static final fun columnConstraintCheck (Ljava/lang/String;Lorg/partiql/ast/expr/Expr;)Lorg/partiql/ast/ddl/AttributeConstraint$Check;
```

Comment:
Taking a look at the api dump, seems like we're also missing DDL support in `AstRewriter` and `SqlDialect`. If those are tackled later, can we add TODOs in those classes?